aws/resource_aws_api_gateway_documentation_part.go

```diff
@@ -0,0 +1,202 @@
+package aws
+
+import (
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/apigateway"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsApiGatewayDocumentationPart() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsApiGatewayDocumentationPartCreate,
+		Read:   resourceAwsApiGatewayDocumentationPartRead,
+		Update: resourceAwsApiGatewayDocumentationPartUpdate,
+		Delete: resourceAwsApiGatewayDocumentationPartDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
```

Comment:
Must fix: missing acceptance test for import functionality

Reply:
Good catch! Will add.